app/src/bcsc-theme/types/navigators.ts

```diff
@@ -205,5 +205,5 @@ export type BCSCMainStackParams = {
 
   [BCSCModals.InternetDisconnected]: undefined
   [BCSCModals.MandatoryUpdate]: undefined
-  [BCSCModals.DeviceInvalidated]: undefined
+  [BCSCModals.DeviceInvalidated]: { caseType?: string }
```

Comment:
suggestion (2): Looks like in the DeviceInvalidated modal that it is expecting a reason why it was invalidated. Possible opportunity to make this param required?